packages/card/test/card.common.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from '@vaadin/chai-plugins';
```

Comment:
We should name this file `card.test.ts` so it actually runs. Also need to import the component:

```js
import '../vaadin-card.js';
```

The structure with `.common` files only needs to be used by components with Polymer and Lit versions.

Reply:
Done